test/e2e/e2e_test.go

```diff
@@ -503,7 +503,7 @@ var _ = Describe("controller", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred(), "Failed to reset nvidia.com/mig-1g.5gb to zero")
 
 			err = deleteInstasliceResources()
-			Expect(err).NotTo(HaveOccurred(), "Failed to delete org.instaslice/* resources")
+			Expect(err).NotTo(HaveOccurred(), "Failed to delete %s* resources", orgInstaslicePrefix)
```

Comment:
Once the constant `orgInstaslicePrefix` is defined,
```suggestion
			Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Failed to delete %s* resources", orgInstaslicePrefix))
```